reproducibility_project/clean_by_engine.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+echo Removing jobs from all engines but $1
```

Comment:
Might be worth adding some check for a valid remove. Either that, or make someone specify exactly what they want to remove. 
i.e. you list the engines you want removed. For lammps-VU, I would remove [lammps-UD, gromacs, mcccs, gomc, hoomd, cassandra]. That way a typo doesn't accidentally delete all of the workspace.

Reply:
yeah I like the idea of having a check with the user. I'll look into how to do this.

Reply:
I dont entirely know if this would work, but it might be one way to accomplish this
You could grep or use signac find on the workspace, if nothing shows up, then error?
https://unix.stackexchange.com/a/223504

Reply:
@CalCraven I implemented your suggestion, so now it prompts the user before deleting anything, which feels much safer imo. :)

Reply:
@justinGilmer The rm command will error if signac find returns nothing, so I don't think it's too dangerous. Putting rm in a script though definitely is a little scary though haha.